Author: Dirk Mueller <dmueller@suse.de>
Subject: Raise default client limit to 512
Patch-Mainline: Never

After a while of starting too many browser tabs, I ran into
xorg: Maximum number of clients reached

which also prevented me from starting a terminal window to debug
the issue.. so raise the default a bit. Based on 

https://plug-discuss.phxlinux.narkive.com/BWkQ7Xez/xorg-maximum-number-of-clients-reached
it appears other distros are also doing that.

Index: xorg-server-21.1.21/include/misc.h
===================================================================
--- xorg-server-21.1.21.orig/include/misc.h
+++ xorg-server-21.1.21/include/misc.h
@@ -88,7 +88,7 @@ OF THIS SOFTWARE.
 #define MAXGPUSCREENS	16
 #endif
 #define MAXCLIENTS	2048
-#define LIMITCLIENTS	256     /* Must be a power of 2 and <= MAXCLIENTS */
+#define LIMITCLIENTS	512     /* Must be a power of 2 and <= MAXCLIENTS */
 #define MAXEXTENSIONS   128
 #define MAXFORMATS	8
 #ifndef MAXDEVICES
Index: xorg-server-21.1.21/man/Xserver.man
===================================================================
--- xorg-server-21.1.21.orig/man/Xserver.man
+++ xorg-server-21.1.21/man/Xserver.man
@@ -324,9 +324,9 @@ A value of zero makes the stack size as
 of \-1 leaves the stack space limit unchanged.
 .TP 8
 .B \-maxclients
-.BR 64 | 128 | 256 | 512
+.BR 64 | 128 | 256 | 512 | 1024 | 2048
 Set the maximum number of clients allowed to connect to the X server.
-Acceptable values are 64, 128, 256 or 512.
+Acceptable values are 64, 128, 256, 512, 1024 or 2048.
 .TP 8
 .B \-render
 .BR default | mono | gray | color
